CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changes
 
+### v0.5.1
+
+* Add `control_features` to the classification moments
```

Comment:
As a user that doesn't really tell me anything. Before releasing v0.5.1 we should at least write an example for how to use this.

Reply:
I'll expand it a bit. There are examples in the user guide, although there isn't a notebook. We really need some decent case studies, rather than manufacturing new examples off the Adult Census data....